src/class/toggleClass.js

```diff
@@ -0,0 +1,16 @@
+import hasClass from './hasClass'
+import removeClass from './removeClass'
+import addClass from './addClass'
+
+export default function toggleClass(element, className, force){
```

Comment:
`force` is sort of a weird argument here...i realize you'r matching the native API, but maybe we don't need to bother with that, and let people use remove/add explicitly when they need too?

Reply:
Thank you. I agree. Done.